StripeCryptoOnramp/StripeCryptoOnramp/Source/Components/CryptoOnrampCoordinator.swift

```diff
@@ -65,6 +65,11 @@ public protocol CryptoOnrampCoordinatorProtocol {
     /// - Parameter walletAddress: The crypto wallet address to register.
     /// - Parameter network: The crypto network for the wallet address.
     func collectWalletAddress(walletAddress: String, network: CryptoNetwork) async throws
+
+    /// Presents the Link sheet to collect a customer's payment method.
+    /// - Parameter viewController: The view controller from which to present the Link sheet.
+    /// - Returns: A `PaymentMethodPreview` if the user selected a payment method, or `nil` otherwise.
+    func collectPaymentMethod(from viewController: UIViewController) async -> PaymentMethodPreview?
```

Comment:
(If this gets posted twice, it's because GitHub was having some issues and failed to post the first time for me.)

The parameter name `viewController` here differs from what's in the API review doc in order to keep consistent with:

```swift
func presentForVerification(from viewController: UIViewController) async throws -> VerificationResult
```

and

```swift
func promptForIdentityVerification(from viewController: UIViewController) async throws -> IdentityVerificationResult
```

If that needs to change, just let me know.